### DISCUSSION TITLE
Suggestion: Announcement tab throughout UI

### DISCUSSION THREAD

**Discussion by edoswald** [mastodon/mastodon#23962] [General]
Currently, Announcements are only shown at the top of the Home page. If a user is elsewhere while an announcement is posted, they will only see it if they return to the Home page. Placing it in the same position by the filter icon at the top of the center column throughout the UI addresses the issue and be useful for those who might use the announcements for urgent messages (unplanned downtime, etc.).

**Comment by scrappydog** [comment]
![image](https://user-images.githubusercontent.com/125875/223731337-b925aa8d-9235-4c96-a5e0-aad37421ab60.png)
In the Advanced interface this tiny announcement notification is almost invisible...

A big RED toast at the top of the page would be way more useful!
